src/ui/forgot_password.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<!--
+  This document should be written in polyglot HTML5/XHTML5, and therefore
+  should run when served either as text/html or application/xhtml+xml.
+-->
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
+<head>
+  <meta charset="utf-8" />
+  <title>Create Account &mdash; Button Men Online</title>
```

Comment:
Did you want to change this title from "Create Account" to something more appropriate?

Reply:
Thanks, fixed.